src/intfuncs.c

```diff
@@ -516,20 +516,22 @@ Obj FuncHASHKEY_BAG(Obj self, Obj obj, Obj opSeed, Obj opOffset, Obj opMaxLen)
   return INTOBJ_INT(HASHKEY_BAG_NC( obj, (UInt4)INT_INTOBJ(opSeed), offs, (int) n));
 }
 
-Int HASHKEY_BAG_NC (Obj obj, UInt4 seed, Int skip, int maxread){
-  #ifdef SYS_IS_64_BIT
+Int HASHKEY_MEM_NC (const void* ptr, UInt4 seed, Int read) {
```

Comment:
If I look at that w/o reading the documentation commnt, it is not clear to me what `read` is about. In particular, it could also be a boolean.
Perhaps `numBytes`, or `bytesToRead`, or just `len` / `size` would be slightly more informative?

Another things: Remove the space before `(`, and put the `{` on a new line (and/or run `clang-format`)